### ISSUE TITLE
[BUG] Inventory not updating in several locations, says character is offline

### ISSUE THREAD

**Issue by Rainabelle** [arkayenro/arkinventory#1736]
**What version number you are using?**
3.10.12

**What game client are you playing? Retail, Wrath, Classic / Live, PTR, Beta**
Wrath

**What language is the game client set to?**
English

**Describe the bug**
Location inventory isn't updating for the mailbox, bank, and guild bank. The location specific window shows the currently played character as [Offline}.

**To Reproduce**
Open character inventory and toggle through the various inventory locations.

**Screenshots**
Attached

**Additional context**
All locations are set to enabled

![image](https://user-images.githubusercontent.com/119996611/206090848-c88cfd21-a218-469b-a249-e3eadf8ee061.png)

**Comment by arkayenro** [comment]
unless you have the mailbox open then the mailbox window will be offline.  same for any window.

**Comment by Rainabelle** [comment]
this occurs when the mailbox or bank is open.

**Comment by arkayenro** [comment]
and arkinventory is the only mod loaded and youre not getting any errors?

**Comment by Rainabelle** [comment]
Yes, I disabled all addons except for arkinventory and I'm still having this issue. My character is showing offline and won't sync my mailbox.
![image](https://user-images.githubusercontent.com/119996611/207168942-5c6ba5c8-6055-4226-83de-a90d3eb52b34.png)
